selene/src/main.rs

```diff
@@ -517,17 +530,23 @@ fn start(mut options: opts::Options) {
                 }
             }
 
-            None => match fs::read_to_string("selene.toml") {
-                Ok(config_contents) => match toml::from_str(&config_contents) {
-                    Ok(config) => (config, None),
-                    Err(error) => {
-                        error!("Config file not in correct format: {}", error);
-                        std::process::exit(1);
-                    }
-                },
+            None => {
+                let read_config_file = |file: &str| -> Option<CheckerConfig<toml::value::Value>> {
+                    let config_contents = fs::read_to_string(file).ok()?;
+                    toml::from_str(&config_contents)
+                        .map_err(|error| {
+                            eprintln!("Error parsing config file {}: {}", file, error);
+                            error
+                        })
+                        .ok()

```

Comment:
Need to early terminate the process when config file is invalid.

Reply:
Closed on https://github.com/Zeioth/selene/commit/de9aa798418e98ee77f8b3d91e8ee865ff66fc49